dcos_test_utils/test_gce.py

```diff
@@ -0,0 +1,78 @@
+import dcos_test_utils.gce as gce
+import dcos_launch
+import yaml
+import os
+import pytest
+
+
+@pytest.fixture(scope='module')
+def gce_wrapper():
+    credentials_path = os.environ.get('GOOGLE_APPLICATION_CREDENTIALS', os.path.expanduser('~') + '/.gce/credentials.json')
+    return gce.GceWrapper(credentials_path)
+
+
+@pytest.fixture(scope='module')
+def config(gce_wrapper):
+    cwd = os.path.dirname(os.path.realpath(__file__))
+    with open(cwd + '/../dcos_launch/sample_configs/gce-onprem.yaml') as cfile:
+        config_content = cfile.read()
+    conf = yaml.load(config_content)
+    template_path = conf.get('template_path', cwd + '/../dcos_launch/templates/gce-template.yaml')
+    with open(template_path, 'r') as tfile:
+        template_content = tfile.read()
```

Comment:
Above was correct, however in this scenario I need the actual text content of the yaml file for the request (and not a dictionary)